modules/ROOT/pages/java/java-invoke-method.adoc

```diff
@@ -54,9 +76,275 @@ You can invoke static Java methods like this:
+.. In *javaexamples*, provide a trigger for a flow by dragging a Scheduler
+component into the Studio canvas.
++
+Optional: You can also set the Scheduler frequency to something less frequent
+than the default, for example, *Frequency* to `10` and *Time unit* to `SECONDS`.
++
+.. If the Java module is not already available in your Mule palette, click
+*Add Module*, and drag the Java module into the left column of the palette.
+.. Click the Java module, place the New operation to the right of the
+Scheduler in the flow, and then double-click and configure the operation:
++
+* *Class*: `com.examples.Hello`
+* *Constructor*: `Hello()`
++
+Do not use the *_fx_* setting for the *Constructor* setting.
++
+.. Place the Invoke operation to the right of the New operation in the flow,
+and then double-click and configure the operation:
++
+* In the *Instance* field, click *_fx_*, and set the value to `payload`.
+* *Class* field: `com.examples.Hello`
+* *Method* field: `hello()`
++
+Do not use the *_fx_* setting for the *Method* setting.
++
+.. Find and drag a Logger component to the right of the Invoke operation in the
+flow, and in its *Message* field, click *_fx_* and type `payload`.
+.. Find and drag a Flow Reference component to the right of the Invoke operation
+in the flow, and then double-click the component and set the *Flow Name* field
+to the name of a new flow that you create in the next step, `javaexamplesFlow1`.
+. Set up a new flow by dragging a new Flow component below the existing flow,
+making sure that its name is `javaexamplesFlow1` (so that your Flow Reference
+setting in the other flow matches the name of the new flow).
+.. Click the Java module, then drag a New operation to the *Process*
+section of your new flow, `javaexamplesFlow1`, and provide the following
+configuration for the operation:
++
+* *Args*: `{ "numA" : 5, "numB" : 6}` for the arguments
+* *Class*: `com.examples.Add`
+* *Constructor*: `Add(int,int)`
++
+Do not use the *fx* setting for the *Constructor* setting.
++
+.. Place an Invoke operation to the right of the New operation in the new flow,
+and then set the following configuration:
++
+* Click *fx*, and set the *Instance* field: `payload`.
+* *Args*: `{ "x" : 3, "y" : 4}` for the arguments to process during the invocation
+* *Class*: `com.examples.Add`
+* *Method*: `add(int,int)`
++
+Do not use the *_fx_* setting for the *Method* setting.
++
+.. Continuing with the New operation, click the *Advanced* configuration link, and
+set the *Output* to a target variable that stores the payload of the Invoke operation:
++
+* *Target Variable*: `mySum`
+* *Target Value*: `payload`
++
+This step simply shows how to pass the payload to a target variable if you
+ever need to do so.
++
+.. Place a Logger component to the right of the Invoke operation in the
+new flow, click *fx*, and set the *Message* field to `vars.mySum`.
++
+This setting prints the payload stored in the target variable to the console.
```

Comment:
I am not sure "print" is the best word here. Maybe "shows the payload...on the console."